eng/cmake/global_compile_options.cmake

```diff
@@ -6,15 +6,22 @@ if(MSVC)
   )
 endif()
 
+if(ADDRESS_SANITIZER)
+  add_compile_options(-fsanitize=address)
+  if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
```

Comment:
For GNU, we need to add both the compile option and link option, or just link option?

Reply:
both. Which is why I add the link option here. I would add it unconditionally but LINK.EXE does not accept it.